### PR TITLE
Added description for new custom static labels

### PR DESCRIPTION
**Pull request by vterentev**
## Checklist

- [x] My article available in Navigation sidebar and in root article of selected category ([how to do it in README](https://github.com/santiment/academy#add-an-article-into-navigation-sidebar))
- [x]	My article have [metadata](https://github.com/santiment/academy#metadata) (title, description, date when updated and author)
- [x]	All added/updated links in article are exist, images shows correctly

### CODE REVIEW

File: src/docs/labels/blocked/index.md [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-description:
+title: Blocked Label
+author: Vladimir
+date: 2023-06-16
+description: Blocked Label
 ---
 
 ## Description
-
+This address has been blocked by custodial stablecoin provider.
```
Comment (alexander-gr) [inline src/docs/labels/blocked/index.md:9]: Would we use this label only for custodial stablecoin providers?

File: src/docs/labels/charity/index.md [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
 
 ## API Name
-
-``
+`charity`
 
 ## Label Examples
-
+Endaoment: [0x00259ef939eb043cd81613f18ebfdb10f250155c](https://etherscan.io/address/0x00259ef939eb043cd81613f18ebfdb10f250155c)
+Endaoment Astraea Foundation: [0xa04ede9f75f54e28c74b1c75d9b44e7418fee4dd](https://etherscan.io/address/0xa04ede9f75f54e28c74b1c75d9b44e7418fee4dd)
```
Comment (alexander-gr) [inline src/docs/labels/charity/index.md:16]: Add blank line between examples

File: src/docs/labels/dao/index.md [hunk 3]
```diff
@@ -1,20 +1,19 @@
...
 
 ## API Name
-
-``
+`dao`
 
 ## Label Examples
-
+Maker Vault Owner: [0xf44220e22573ac6557a60d1973d2a088bac8dd22](https://etherscan.io/address/0xf44220e22573ac6557a60d1973d2a088bac8dd22)
+Maker Old Contract: [0xc66ea802717bfb9833400264dd12c2bceaa34a6d](https://etherscan.io/address/0xc66ea802717bfb9833400264dd12c2bceaa34a6d)
```
Comment (alexander-gr) [inline src/docs/labels/dao/index.md:16]: Discuss

File: src/docs/labels/derivatives/index.md [hunk 4]
```diff
@@ -1,19 +1,25 @@
...
 
 ## API Name
 
-``
+`derivatives`
 
 ## Label Examples
-
+Synthetix Exchange State [0x545973f28950f50fc6c7f52aab4ad214a27c0564](https://etherscan.io/address/0x545973f28950f50fc6c7f52aab4ad214a27c0564)
+Synthetix Token State sEUR [0x6568d9e750fc44af00f857885dfb8281c00529c4](https://etherscan.io/address/0x6568d9e750fc44af00f857885dfb8281c00529c4)
```
Comment (alexander-gr) [inline src/docs/labels/derivatives/index.md:22]: Add blank line

File: src/docs/labels/eth2-staking/index.md [hunk 5]
```diff
@@ -7,9 +7,7 @@ description: ETH2 Staking label
 
 ## Description
 
-The eth2_staking label is used to identify wallet addresses that are actively participating in the Ethereum 2.0 staking process. Ethereum 2.0 is an upgrade to the Ethereum network that transitions it from a proof-of-work (PoW) to a proof-of-stake (PoS) consensus mechanism.
-
-By applying the eth2_staking label to specific addresses, you can quickly identify wallets that are involved into staking activities. Staking involves locking up a certain amount of ETH as a validator in the Ethereum network, participating in the block validation process, and earning rewards in return for helping to secure the network.
+The eth2_staking label is used to identify wallet addresses that are used to put eth on staking
```
Comment (alexander-gr) [inline src/docs/labels/eth2-staking/index.md:10]: Lets refactor to `wallet addresses that stake ETH`

File: src/docs/labels/yield/index.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+## Description
+Label used to mark addresses that are used in Yield Farming activities
+
+## API Name
+
+`yield`
+
+## Label Examples
+YFV.finance: Link Pool: [0x196CF719251579cBc850dED0e47e972b3d7810Cd](https://etherscan.io/address/0x196cf719251579cbc850ded0e47e972b3d7810cd)
+Pylon.Finance: Solarite SNX Pool: [0x92a7492A1354dFFB9A9865B66e8334736eF49374](https://etherscan.io/address/0x92a7492a1354dffb9a9865b66e8334736ef49374)
```
Comment (alexander-gr) [inline src/docs/labels/yield/index.md:17]: Blank line

**Review by alexander-gr — COMMENTED** [review]
TODO: Update navigation page

File: src/docs/labels/dao/index.md [hunk 7]
```diff
@@ -1,19 +1,18 @@
 ---
-title: DAO Label
+title:
```
Comment (IvanIvanoff) [inline src/docs/labels/dao/index.md]: missing title?

Comment (vterentev) [reply src/docs/labels/dao/index.md]: Just in general decided to remove current info about this label (and exclude from labels index)

**Review by alexander-gr — APPROVED** [review]

**Pull request merged**